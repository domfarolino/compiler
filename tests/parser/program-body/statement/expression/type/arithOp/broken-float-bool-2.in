program dominic is

  float result;

begin

  result := 10.33 + (10.42 != 5 * 2);

end program.
