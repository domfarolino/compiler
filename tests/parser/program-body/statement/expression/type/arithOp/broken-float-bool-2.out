[1;31mLine 7: [ArithOp]: float and bool cannot be added or subtracted
Line 7: Error parsing program body
[0m